tools/testoperator/README.md

```diff
@@ -20,6 +20,7 @@ testoperator run [COMMAND] [OPTIONS]
 - `load`: Run a load test.
 - `bench`: Run a benchmark test.
 - `data-consistency`: Run a data consistency test.
+- `http-consistency`: Runs a test to compare the latency performance of a HTTP enabled component as the component is persistently queried.
```

Comment:
nit: Add the `http-overhead` command here

Reply:
added